src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2937,7 +2937,7 @@ components:
   schemas:
     TableDataType:
       type: string
-      enum: [ boolean, bytes, date, datetime, dirref, fileref, float, float64, integer, int64, numeric, record, string, text, time, timestamp ]
+      enum: [ fileref, boolean, bytes, date, datetime, dirref, float, float64, integer, int64, numeric, record, string, text, time, timestamp ]
```

Comment:
any reason this is changed?  Kind of a bummer to loose alphabetical order

Reply:
my reasoning was that the example schema in our swagger only shows the first value of the enum:
"schema": {
    "tables": [
      {
        "name": "string",
        "columns": [
          {
            "name": "string",
            "datatype": "boolean",
            "array_of": false
          }
        ],
I find datatype: boolean to be confusing since the user might think it should be a true/false value. 
On second thought I might want to change it to datatype: string.